core/src/jvmMain/kotlin/arrow/endpoint/CodecUtils.kt

```diff
@@ -0,0 +1,74 @@
+package arrow.endpoint
+
+import arrow.endpoint.model.CodecFormat
+import java.io.InputStream
+import java.math.BigDecimal
+import java.nio.ByteBuffer
+import java.time.Duration as JavaDuration
+import java.time.Instant
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+import java.time.OffsetDateTime
+import java.time.OffsetTime
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeParseException
+import java.util.UUID
+import java.util.Date
+
+public val Codec.Companion.uuid: Codec<String, UUID, CodecFormat.TextPlain>
+  get() = stringCodec(Schema.uuid, UUID::fromString)
```

Comment:
Cannot be `=` without `get()`, right? Can we cache the instances?

Reply:
<img width="901" alt="Screenshot 2022-01-24 at 17 13 43" src="https://user-images.githubusercontent.com/46971368/150820669-06749b9a-a531-42a2-a3db-f11d9b2fbf9f.png">

Reply:
I haven't looked into Caching, yet ☺️